### ISSUE TITLE
Broken by new WTForms release

### ISSUE THREAD

**Issue by aireenmei** [dpgaspar/Flask-AppBuilder#2276]
WTForms released 3.2.0 today. Get the following error. Downgrade WTForms to 3.1.2 works.
```
    from .fields import EnumField, QuerySelectField, QuerySelectMultipleField
  File "/usr/local/google/home/aireenmei/venv/airflow/lib/python3.11/site-packages/flask_appbuilder/fields.py", line 8, in <module>
    from wtforms.fields import Field, SelectField, SelectFieldBase
ImportError: cannot import name 'SelectFieldBase' from 'wtforms.fields' (xxxx/python3.11/site-packages/wtforms/fields/__init__.py)
```

**Comment by raimundovidaljunior** [comment]
yup, facing the same issue here

**Comment by Tenzer** [comment]
There's a new 3.2.1 release out of wtforms which supposedly fixes this: https://github.com/pallets-eco/wtforms/releases/tag/3.2.1.

**Comment by potiuk** [comment]
Hello @dpgaspar unfortunately even WTForms 3.2.1 that are fixiing the problem with import do not fix another problems with WTForms 3.2.

I raised another issue as our tests started to fail and it looks like Flask Appbuilder uses deprecated tuple flags https://github.com/pallets-eco/flask-wtf/issues/608#issuecomment-2428414572 that have been removed in WTForms 3.2 so WTForms 3.2 are incompatible with the currently released Flask Appbuilder. 

I guess it calls for new release of FAB.
